Detections/MultipleDataSources/BariumDomainIOC.yaml

```diff
@@ -0,0 +1,125 @@
+id: 70b12a3b-4899-42cb-910c-5ffaf9d7997d 
+name: Known Barium domains  
+description: | 
+  'Identifies a match across various data feeds for domains IOCs related to the Barium activity group.' 
+   References: https://www.justice.gov/opa/pr/seven-international-cyber-defendants-including-apt41-actors-charged-connection-computer'
```

Comment:
Don't think you need the ' at the end of the first line - worried this might break something.